msg/ModeCompleted.msg

```diff
@@ -1,4 +1,5 @@
-# Mode completion result, published by an active mode.
+# Mode completion result, published by an active mode. The possible values of nav_state are defined 
+# in the [source VehicleStatus](https://github.com/PX4/PX4-Autopilot/blob/main/msg/VehicleStatus.msg).
```

Comment:
I think first sentence gets used for the one line description, so slightly ordering.
```suggestion
# Mode completion result, published by an active mode.
# The possible values of nav_state are defined in the 
# [source VehicleStatus](https://github.com/PX4/PX4-Autopilot/blob/main/msg/VehicleStatus.msg).
```